### ISSUE TITLE
Repository organization

### ISSUE THREAD

**Issue by eatdrinksleepcode** [adamralph/bau#213]
The XUnit task is currently in the main repository, while the MSBuild task is in its own repository. I like the idea of keeping MSBuild in its own repository, since this encourages thinking of Bau as a task runner on top of which a build system can be constructed. However, if that is the goal, it feels odd that XUnit is in the main repo (all the more so since it is unlikely that many people will use XUnit without MSBuild).

**Comment by adamralph** [comment]
Agreed. IIRC it was my intention to move it out but never got round to it.
